README.md

```diff
@@ -132,18 +132,54 @@ All classes are considered internal and are not subject to semver.
 
 ## Installation
 
-The recommended installation method is through [Composer](https://getcomposer.org/).
-Simply run
-
-    composer require felixfbecker/language-server
-
-and you will get the latest stable release and all dependencies.  
-Running `composer update` will update the server to the latest non-breaking version.
-
-After installing the language server and its dependencies,
-you must parse the stubs for standard PHP symbols and save the index for fast initialization.
-
-     composer run-script --working-dir=vendor/felixfbecker/language-server parse-stubs
+The recommended installation method is through [Composer](https://getcomposer.org/).  Follow the following installation instructions for local or global installation after installing composer on your system.
+
+### Local Installation
+Create a directory for php-language-server. Create a composer.json file in it, with the following contents:
```

Comment:
I believe the idea here is that some users will want to try this out without making any changes to their global composer config, and these instructions provide a way of creating a local (perhaps temporary) instance of the language server without touching anything else.

The text editor can then be pointed to the local installation for testing.  If it all pans out, the user might move to a global install (or not, as the case may be).

So "local" installation here can be thought of as a self-contained installation, rather than integrating it into another project like vscode-php-intellisense.  Perhaps that should be covered as a third case, though?

Reply:
An example of why users might want a separate/self-contained installation is so that they can patch the language server for specific use-cases without affecting their global installation.  For instance, modifying the hard-coded set of filename extensions which should be treated as PHP files in a particular project type -- a need that I personally ran into yesterday.